[2.x] panic in testnet at block 5899351

```
panic: bad integer length

goroutine 58 [running]:
github.com/nspcc-dev/neo-go/pkg/core/state.(*NEP5Transfer).EncodeBinary(0xc00c62c400, 0xc00c62c500)
        github.com/nspcc-dev/neo-go/pkg/core/state/nep5.go:219 +0x356
github.com/nspcc-dev/neo-go/pkg/core/state.(*TransferLog).Append(0xc00c61a438, 0xff40f8, 0xc00c62c400, 0x1c0684bb3b8, 0xc00c61a438)
        github.com/nspcc-dev/neo-go/pkg/core/state/nep5.go:111 +0x16c
github.com/nspcc-dev/neo-go/pkg/core/dao.(*Cached).AppendNEP5Transfer(0xc0005f7620, 0x8108a57ca4b606c8, 0xee3a4d0937a6fdfd, 0x1c0684bb3b8, 0xc00c62c400, 0x0, 0x0, 0x7feb6ba81060)
        github.com/nspcc-dev/neo-go/pkg/core/dao/cacheddao.go:210 +0x99
github.com/nspcc-dev/neo-go/pkg/core.(*Blockchain).processNEP5Transfer(0xc000224240, 0xc0005f7620, 0xc00c62c400)
        github.com/nspcc-dev/neo-go/pkg/core/blockchain.go:960 +0x5a9
github.com/nspcc-dev/neo-go/pkg/core.(*Blockchain).storeBlock(0xc000224240, 0xc00c5a6000, 0x0, 0xd6b100)
        github.com/nspcc-dev/neo-go/pkg/core/blockchain.go:809 +0x200e
github.com/nspcc-dev/neo-go/pkg/core.(*Blockchain).AddBlock(0xc000224240, 0xc00c5a6000, 0x0, 0x0)
        github.com/nspcc-dev/neo-go/pkg/core/blockchain.go:444 +0xeb
github.com/nspcc-dev/neo-go/pkg/network.(*blockQueue).run(0xc00c0cd5c0)
        github.com/nspcc-dev/neo-go/pkg/network/blockqueue.go:48 +0x168
created by github.com/nspcc-dev/neo-go/pkg/network.(*Server).Start
        github.com/nspcc-dev/neo-go/pkg/network/server.go:183 +0x25d
```
Not sure if it's particular node gone wild or something more serious.

--- a/pkg/core/blockchain.go
+++ b/pkg/core/blockchain.go
@@ -945,15 +945,14 @@ func (bc *Blockchain) processNEP5Transfer(cache *dao.Cached, transfer *state.NEP
 			return
 		}
 		bs := balances.Trackers[transfer.Asset]
-		if bs.Balance == nil {
-			return
-		}
-		bs.Balance.Sub(bs.Balance, transfer.Amount)
-		if bs.Balance.Sign() > 0 {
-			bs.LastUpdatedBlock = transfer.Block
-			balances.Trackers[transfer.Asset] = bs
-		} else {
-			delete(balances.Trackers, transfer.Asset)
+		if bs.Balance != nil {
+			bs.Balance.Sub(bs.Balance, transfer.Amount)
+			if bs.Balance.Sign() > 0 {
+				bs.LastUpdatedBlock = transfer.Block
+				balances.Trackers[transfer.Asset] = bs
+			} else {
+				delete(balances.Trackers, transfer.Asset)
+			}
 		}
 
 		transfer.Amount.Neg(transfer.Amount)

--- a/pkg/core/state/nep5.go
+++ b/pkg/core/state/nep5.go
@@ -188,6 +188,9 @@ func NEP5TransferFromNotification(ne NotificationEvent, txHash util.Uint256, hei
 		if !ok {
 			return nil, errors.New("wrong amount type")
 		}
+		if len(bs) > amountSize {
+			return nil, errors.New("integer overflow")
+		}
 		amount = emit.BytesToInt(bs)
 	}
 	toAddr := parseUint160(to)

